packages/main/src/themes/TableHeaderRow.css

```diff
@@ -1,6 +1,5 @@
 :host {
     background: var(--sapList_HeaderBackground);
-    grid-template-rows: var(--_ui5_list_item_base_height);

```

Comment:
Is there a specific reason, why this has been removed?
If I remember correctly the reason for adding this, was to avoid changing header row heights when switching selection modes